<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Drawer3D</class>
 <widget class="QMainWindow" name="Drawer3D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>565</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3D Drawer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="runge.qrc">
    <normaloff>:/Runge/Resources/16x16/view.png</normaloff>:/Runge/Resources/16x16/view.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutAll">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutTop">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="labelX">
          <property name="text">
           <string>&amp;X</string>
          </property>
          <property name="buddy">
           <cstring>xAxisBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="xAxisBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>52</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>52</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>X axis variable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelY">
          <property name="text">
           <string> &amp;Y</string>
          </property>
          <property name="buddy">
           <cstring>yAxisBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="yAxisBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>52</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>52</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Y axis variable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelZ">
          <property name="text">
           <string> &amp;Z</string>
          </property>
          <property name="buddy">
           <cstring>zAxisBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="zAxisBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>52</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>52</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Z axis variable</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonPan">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>24</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Pan</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="runge.qrc">
            <normaloff>:/Runge/Resources/16x16/palm_hold.png</normaloff>:/Runge/Resources/16x16/palm_hold.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSettings">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Drawer Settings</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="runge.qrc">
            <normaloff>:/Runge/Resources/16x16/wrench.png</normaloff>:/Runge/Resources/16x16/wrench.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonExport">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Export to File</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="runge.qrc">
            <normaloff>:/Runge/Resources/16x16/export1.png</normaloff>:/Runge/Resources/16x16/export1.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonHelp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>24</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Help</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="runge.qrc">
            <normaloff>:/Runge/Resources/16x16/help.png</normaloff>:/Runge/Resources/16x16/help.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutBottom">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutSlidersAngles">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutSliders">
            <item>
             <widget class="QSlider" name="xSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>X rotation angle</string>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="ySlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Y rotation angle</string>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="zSlider">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Z rotation angle</string>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout" columnstretch="0,0" columnminimumwidth="0,50">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>2</number>
            </property>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>rot &amp;Z</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>zAngleSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>rot &amp;Y</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>yAngleSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>rot &amp;X</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>xAngleSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="zoomSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Zoom Factor</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="decimals">
               <number>3</number>
              </property>
              <property name="minimum">
               <double>0.001000000000000</double>
              </property>
              <property name="maximum">
               <double>1000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.001000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Zoo&amp;m</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>zoomSpinBox</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="zAngleSpinBox">
              <property name="toolTip">
               <string>Z rotation angle</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>360.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="yAngleSpinBox">
              <property name="toolTip">
               <string>Y rotation angle</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>360.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="xAngleSpinBox">
              <property name="toolTip">
               <string>X rotation angle</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>360.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>xAxisBox</tabstop>
  <tabstop>yAxisBox</tabstop>
  <tabstop>zAxisBox</tabstop>
  <tabstop>buttonPan</tabstop>
  <tabstop>buttonSettings</tabstop>
  <tabstop>buttonExport</tabstop>
  <tabstop>buttonHelp</tabstop>
  <tabstop>xSlider</tabstop>
  <tabstop>ySlider</tabstop>
  <tabstop>zSlider</tabstop>
  <tabstop>xAngleSpinBox</tabstop>
  <tabstop>yAngleSpinBox</tabstop>
  <tabstop>zAngleSpinBox</tabstop>
  <tabstop>zoomSpinBox</tabstop>
 </tabstops>
 <resources>
  <include location="runge.qrc"/>
 </resources>
 <connections/>
</ui>
